webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/DataStructures.scala

```diff
@@ -357,7 +357,7 @@ final case class Policy(
   // == .toList (keep order)     ==> List[List[Variable]]
   // == flatten (keep order)     ==> List[Variable]
   val expandedVars    = Policy.mergeVars(policyVars.map( _.expandedVars.values).toList.flatten)
-  val originalVars    = Policy.mergeVars(policyVars.map( _.originalVars.values).toList.flatten)
+  //val originalVars    = Policy.mergeVars(policyVars.map( _.originalVars.values).toList.flatten)
```

Comment:
this one seems to have a lot of impact
Size deepof nodeConfigs is 2 668 075 072 
Size deepof ruleVals is 376 459 952

vs
Size deepof nodeConfigs is 3 474 356 896
Size deepof ruleVals is 748 776 080 

but we need to validate as there is a lot of variability in ruleval (from 748 487 288 to 2 123 635 544)

Reply:
nodeconfig is stable at about 2 668 075 048 with this line vs 3 474 356 896 without this line